meta-mbl-distro/recipes-security/ecryptfs-utils/files/ecryptfs-init.sh

```diff
@@ -0,0 +1,250 @@
+#!/bin/bash
+#
+###############################################################################
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# ecryptfs homepage: http://ecryptfs.org/
+# Summary: Utilities for the Linux cryptographic filesystem ecryptfs.
+#
+# Description
+# ===========
+#  This script is used to mount an ecryptfs ciphered directory at system
+#  startup. If the ciphered directory is being mounted for the first time
+#  (e.g. when the system boots after flashing an image) then the required
+#  configuration is generated and stored. The configuration includes:
+#  - An ecyptfs FEKEK (File Encryption Key Encryption Key).
+#  - A passphrase to protect the FEKEK, and the storing of the passphrase
+#    in a file.
+#  - An entry in /etc/fstab specifying the mount operation. This uses the
+#    passphrase file.
+#  - The creation of the upper (unencrypted) and lower (encrypted) storage
+#    directories.
+#
+#  The following operations happen during boot:
+#  - The passphrase protected FEKEK is added to the user kernel keyring.
+#  - The unecrypted (upper) directory is mounted on the encrypted (lower)
+#    directory by ecryptfs to provide deciphered access to store files.
+#    Mount options in fstab specify the FEKEK key signature so ecryptfs
+#    can read the cipher key from the keyring. The passphrase is also read
+#    to decode the FEKEK.
+#
+#  The upper and lower directory terminology is explained in:
+#    <src/>/Documentation/filesystems/overlayfs.txt
```

Comment:
I can not find this path  /Documentation/filesystems/overlayfs.txt in meta-mbl repo. Could you give an exact location ?

Reply:
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/filesystems/overlayfs.txt